### PR TITLE
improve transport modal

### PR DESCRIPTION
**Pull request by jessiesiat**
Signed-off-by: Jessie Siat som_grad@yahoo.com

This is an improvement fix, the one which causes the site http://bukaspalad.appnimbus.com/dashboard broken.
That is trying to pack a donation without available package. 

This is just to not let them from submitting the form.

### CODE REVIEW

File: application/views/plugins/repack_modal.blade.php [hunk 1]
```diff
@@ -10,21 +10,29 @@
...
-                    @if ($package->is_transported != 1)
-                    <option value="{{$package->id}}">{{$package->area}} (Package #{{$package->id}})</option>
-                    @endif
-                    @endforeach
-                </select>
-            </div>
+            @if((Package::where('is_transported', '=', '0')->count()) == 0)
+                <div class="alert alert-info">Oops! No package available, create new one.</div>
+            @else
+                <?php $package = 1 ?>
```
Comment (nikkobautista) [inline application/views/plugins/repack_modal.blade.php]: You might need to select a different variable name other than $package, since the @foreach loop here will replace the $package variable.

File: application/views/plugins/repack_modal.blade.php [hunk 2]
```diff
@@ -10,21 +10,29 @@
...
+                    </select>
+                </div>
+            @endif
+
         </form>
     </div>
     <div class="modal-footer">
         <button class="btn" data-dismiss="modal" aria-hidden="true">Cancel</button>
-        <button class="btn btn-primary" data-loading-text="Posting..." onclick="$('#repack_modal_form').submit()">Mark as Repacked</button>
+        @if( isset($package) and ($package == 1))
```
Comment (nikkobautista) [inline application/views/plugins/repack_modal.blade.php]: This might never occur since $package is replaced by the foreach loop on line 20.

File: application/views/plugins/transport_modal.php [hunk 3]
```diff
@@ -1,22 +0,0 @@
```
Comment (nikkobautista) [inline application/views/plugins/transport_modal.php:-1]: Why did you remove this whole file?

**Comment by jessiesiat** [comment]
Opps.. I have it remove while changing the extension to .blade.php 
Sorry for that, it still exist it is just the extension that change
